cloud-formation/scripts/post-dev-stack-creation.sh

```diff
@@ -3,6 +3,8 @@
 source ./utils.sh
 source ./stack-name.sh
 
+export AWS_PROFILE=media-service
```

Comment:
would this need to go above `source ./stack-name.sh` so that its set before any call to the awscli is made?